test/unit/actions/action_with_sub_plans_test.rb

```diff
@@ -31,6 +32,8 @@ def run; end
     end
 
     describe Actions::ActionWithSubPlans do
+      include ForemanTasks::TestHelpers::WithInThreadExecutor
```

Comment:
We were using `trigger` with the default executor: this got broken once the default hook was introduced